pom.xml

```diff
@@ -39,7 +39,7 @@
         <docker.image.prefix>mszarlinski</docker.image.prefix>
         <docker.image.exposed.port>9090</docker.image.exposed.port>
         <docker.image.dockerfile.dir>${basedir}</docker.image.dockerfile.dir>
-        <docker.plugin.version>0.4.13</docker.plugin.version>
+        <docker.plugin.version>1.1.1</docker.plugin.version>
```

Comment:
there was an attempt to fix the non-breaking docker build exception, didn't work, but I figured why not just keep it on the latest.